### ISSUE TITLE
[Service Manual] Accessibility and Privacy Statement

### ISSUE THREAD

**Issue by yatesn** [ONSdigital/design-system#2702]
Both are currently missing and need to be added to the Service Manual. 

Accessibility statement potentially blocked until next DAC assessment (date pending)

Adding to this repo for visibility

**Comment by emilymaylawes** [comment]
Relevant trello tickets:

https://trello.com/c/buoRldNb/593-epic-service-manual-footer-pages - Epic

https://trello.com/c/pBO6pnP0/596-service-manual-accessibility-statement
https://trello.com/c/SJJVW6P5/598-service-manual-terms-and-conditions
https://trello.com/c/vbGrhSZ1/597-service-manual-data-protection

**Comment by emilymaylawes** [comment]
Accessibility statement ticket https://github.com/orgs/ONSdigital/projects/21/views/1?pane=issue&itemId=32860220

**Comment by emilymaylawes** [comment]
Privacy policy needs to go on

**Comment by yatesn** [comment]
Set up meeting with Oscar/Dina(?)/Precious
Check legal contact
